### DISCUSSION TITLE
Source Mirror is gone from OBS

### DISCUSSION THREAD

**Discussion by StoneyTaboney** [Xaymar/obs-stream-effects#916] [Q&A] [answered]
Can anyone help me with this? Source mirror was a huge part of my OBS and now I have spout which is not the same. No matter now many times I uninstall and re-install I still don't have it.

**Comment by Xaymar** [comment, marked as answer]
For OBS Studio 28.0.0 and beyond, you will have to opt for v0.12.x and newer. For OBS Studio 27.2.4 you will have to opt for v0.11.1. Ensure that you've picked the correct installation location, and then it should work out of the box - assuming your system meets the System Requirements.

**Reply by StoneyTaboney** [reply]
I updated OBS then rolled back. I have followed all steps to install and my system meets reqs. Should I uninstall and reinstall it all?

**Reply by Xaymar** [reply]
Yes. Ensure that you've used a version compatible with the OBS Studio version of your choice.

**Reply by StoneyTaboney** [reply]
Went ahead and wiped everything. reinstalled and it is now working. Thank you for the reply. Much appreciated!
